web/src/buddy_list.ts

```diff
@@ -42,8 +42,8 @@ function get_total_human_subscriber_count(
     current_sub: StreamSubscription | undefined,
     pm_ids_set: Set<number>,
 ): number {
-    // Excludes human users, but may include long-inactive users who
-    // might not show up in the buddy list.
+    // Excludes bots, but may not include long-inactive users who
+    // don't show up in the buddy list.
```

Comment:
I think `peer_data.get_subscriber_count` will include long-inactive users; I think only the bots part was wrong.

Reply:
I'll actually just remove the whole comment -- "excludes bots" is obvious from the function name